File: KingdomCodeTool/wwwroot/Download/202410291354/Service/Implement/DanhMucBenhVienService.cs
namespace Service.Implement
{
    public class DanhMucBenhVienService : BaseService<DanhMucBenhVien, IDanhMucBenhVienRepository>
    , IDanhMucBenhVienService
    {
    private readonly IDanhMucBenhVienRepository _DanhMucBenhVienRepository;
    public DanhMucBenhVienService(IDanhMucBenhVienRepository DanhMucBenhVienRepository) : base(DanhMucBenhVienRepository)
    {
    _DanhMucBenhVienRepository = DanhMucBenhVienRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202410291354/Angular/shared/DanhMucBenhVien.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucBenhVien } from 'src/app/shared/DanhMucBenhVien.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucBenhVienService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucBenhVien";
    }
}

